rotkehlchen/tests/api/test_balances.py

```diff
@@ -335,7 +335,7 @@ def test_query_all_balances_ignore_cache(
                 assert fn.call_count == 2, msg
         msg = 'etherscan call count should have doubled after forced token detection'
         # TODO: Figure out a correct formula for this
-        expected_count = full_query_etherscan_count * 2 - 2
+        expected_count = full_query_etherscan_count * 2 - 1
```

Comment:
Here I have doubts because I'm not totally sure how the tests for this part work. Now that I removed one query for veCRV there is still a multicall query for it https://github.com/rotki/rotki/blob/bugfixes/rotkehlchen/tests/utils/blockchain.py#L313 so it's off by 1 but I'm not sure what triggers that multicall

Reply:
Is the multicall query for veCRV or for all tokens?

Reply:
Is true. Seems that never asks for the `veCRV` multicall. So I don't know why there was a `-2` in the formula